src/rust/generic-subgraph-generator/test_data/events.json

```diff
@@ -0,0 +1,73 @@
+[{
```

Comment:
We attempt to use every implemented node type in generic-subgraph-generator in this example log. It tells the story of a process (`example_parent.exe`), spawning a child process (`example.exe`), which then reads sensitive data from Chrome, writes a windows batch file (`run_me.bat`), executes the file which then reaches out to an external IP address, and that IP address then calls back to the original child process (`example.exe`).

Just a note regarding `compressed_events.zstd`:

`compressed_events.zstd` is generated from this file by running `zstd events.json -o compressed_events.zstd`.

Reply:
💯